src/loan_form/loan_entry/LoanEntry.jsx

```diff
@@ -5,14 +5,11 @@ import RemoveLoan from '../buttons/RemoveLoanButton'
 export default class LoanEntry extends React.Component {
   constructor (props) {
     super(props)
-    this.state = this.props
     this.handleChange = this.handleChange.bind(this)
   }
 
-  handleChange(event) {
-    this.setState({
-      [event.target.name]: event.target.value
-    })
+  handleChange(e) {
```

Comment:
you can destructure this here: `handleChange({ target })` or even, if you feel brave, `handleChange({ target: { id, value, name } })`